src/redux/books/books.js

```diff
@@ -18,14 +29,13 @@ export function removebook(id) {
   };
 }
 
-export default function bookreducer(state = initialState, action = {}) {
+export default function bookreducer(state = initialState, action) {
   switch (action.type) {
     case AddBook:
       return state.concat(action.payload);
     case RemoveBook:
       return {
-        ...state,
-        payload: state.filter((book) => book.id !== action.payload),
+        ...state.filter((book) => book.id !== action.payload),
       };
     default:
       return state;
```

Comment:
- [ ] [Optional] Everything is fine, but when I try to remove the added data then It shows a white screen and I have to refresh the page. Please fix this issue.

Reply:
ok i will thanks for reviewing it